apps/verticalhome/js/app.js

```diff
@@ -126,10 +132,19 @@
             window.dispatchEvent(new CustomEvent('moz-app-loaded'));
           });
       }.bind(this));
+
+      this.pinManager = new PinAppManager();
+      navigator.mozL10n.ready(this.l10nInit.bind(this));
```

Comment:
We add event listener for timeformatchange event where we refresh time format and locale also.
I check this case. Localization for clock on homescreen changes.